src/main/java/com/bynder/sdk/model/HttpConnectionSettings.java

```diff
@@ -0,0 +1,82 @@
+package com.bynder.sdk.model;
+
+import okhttp3.Interceptor;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.X509TrustManager;
+
+/**
+ * Configuration holder for http connection related settings.
+ */
+public class HttpConnectionSettings {
+
+    public static final int DEFAULT_TIMEOUT_SECONDS = 30;
+
+    /**
+     * SSL Context for the connection to Bynder.
+     * Allows for example to send a client SSL certificate.
+     * Can only be used if a trustManager is provided.
```

Comment:
Here you can remove the reference to Bynder, if you add it to the javadoc of the class like I say in my comment above this one.
Change this one to:
/**
 * SSL Context: allows to send a client SSL certificate.
 * Can only be used if the trustManager was defined.
 */